__tests__/commands/_helpers.js

```diff
@@ -28,6 +28,18 @@ export const runInstall = run.bind(
   [],
 );
 
+export const runInstallWithoutCheck = run.bind(
```

Comment:
Since the `optional-failing` module will be removed from `node_modules`, `check` will fail.